app/views/posts/index.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Display all posts</h1>
\ No newline at end of file
+<img src="<%= asset_path('im12.jpg') %>" alt="User Photo">
+<div class="specificUser">
+<h2><%= @user.name %> </h2>
+<p>Number of Posts: <%= @user.posts.count %></p>
+</div>
+
+<% @posts.each_with_index do |post, index| %>
+  <div class="userPost">
+    <h2>Post #<%= index + 1 %></h2>
+    <a class ="postclick" href="<%= user_post_path(@user, post) %>">
+      <p><%= truncate(post.text, length: 50, separator: ' ') %></p>
+    </a>
+    <p>Comments: <%= post.comments.count %>, Likes: <%= post.likes.count %></p>
+  </div>
+  <div class="userCom">
+  <% post.recent_comments.each do |comment| %>
+            <% if comment.user.present? %>
+              <p><%= comment.user.name %>: <%= comment.text %></p>
+            <% else %>
+              <p>Anonymous: <%= comment.text %></p>
+            <% end %>
+          <% end %>
+  
+  </div>
+<% end %>
```

Comment:
- [ ] Just as I have suggested above, You need to make sure your design follows the given wireframe, Your current design doesn't follow the layout completely, I'll suggest you update this correctly. :+1: :100: 

- - Your current Design:
![Selection_041](https://github.com/faisalakb/Rails-Blog-App/assets/107632780/37e103a8-d72e-44d9-b7c7-31fc22b4ac70)

- - Wireframe:
![Screenshot from 2023-11-24 09-26-10](https://github.com/faisalakb/Rails-Blog-App/assets/107632780/755f1c66-9350-4d7f-8d06-3e9a010125c0)

- [ ] Also as I suggested above this `<img src="<%= asset_path('im12.jpg') %>" alt="User Photo">` is not totally correct. The user's photo is meant to be from the user's photo link in the database and not a hardcoded image file. :+1: :100: